TeamCode/src/main/java/org/firstinspires/ftc/teamcode/localization/TwoDeadWheelLocalizer.java

```diff
@@ -13,7 +15,6 @@
 import com.acmerobotics.roadrunner.ftc.PositionVelocityPair;
 import com.acmerobotics.roadrunner.ftc.RawEncoder;
 import com.qualcomm.robotcore.hardware.DcMotorEx;
-import com.qualcomm.robotcore.hardware.DcMotorSimple;
```

Comment:
Why was this import removed? This is needed to reverse encoders

Reply:
marked as unused

Reply:
I agree with j5155; see the other thread on this import